com.unity.render-pipelines.core/Editor/Volume/Drawers/TextureParameterDrawer.cs

```diff
@@ -0,0 +1,132 @@
+using System;
+using System.Reflection;
+using UnityEngine;
+using UnityEngine.Rendering;
+using Object = UnityEngine.Object;
+
+namespace UnityEditor.Rendering
+{
+    internal static class TextureParameterHelper
+    {
+        static readonly Type k_ObjectFieldValidator = Type.GetType("UnityEditor.EditorGUI+ObjectFieldValidator,UnityEditor");
+        static readonly MethodInfo k_ValidateObjectFieldAssignment = Type.GetType("UnityEditor.EditorGUI,UnityEditor").GetMethod("ValidateObjectFieldAssignment", BindingFlags.Static | BindingFlags.NonPublic);
+        static readonly MethodInfo k_EditorGUI_DoObjectField = Type.GetType("UnityEditor.EditorGUI,UnityEditor")
+            .GetMethod("DoObjectField", BindingFlags.Static | BindingFlags.NonPublic, null,
+                    new Type[] { typeof(Rect), typeof(Rect), typeof(int), typeof(Object), typeof(Object), typeof(Type), typeof(Type), typeof(SerializedProperty), k_ObjectFieldValidator, typeof(bool), typeof(GUIStyle) }, null);
+
+        internal delegate Object ObjectFieldValidator(Object[] references, Type objType, SerializedProperty property, int options);
+
+        internal static Object ValidateObjectFieldAssignment(Object[] references, Type objType, SerializedProperty property, int options)
+        {
+            return (Object)k_ValidateObjectFieldAssignment.Invoke(null, new object[] { references, objType, property, options });
```

Comment:
[Performence] You should compile once using linq tree at static construction instead of researching the method on the object each time with Invoke. Here it is a static one. Not 100% sure there is perf to gain beside the GC of the object array.

Reply:
actually it's the same issue as the one below for this function, I reverted the change